app/Livewire/SettingsOauth.php

```
@@ -38,16 +47,48 @@ public function mount()
+                'client_secret' => $oauth->client_secret,
+                'redirect_uri' => $oauth->redirect_uri,
+                'tenant' => $oauth->tenant,
+                'base_url' => $oauth->base_url,
+            ];
+
             $this->dispatch('success', 'OAuth settings for '.$oauth->provider.' updated successfully!');
         } else {
-            foreach (array_values($this->oauth_settings_map) as &$setting) {
-                $setting->save();
+            foreach (array_values($this->oauth_settings_map) as $settingData) {
+                $oauth = OauthSetting::find($settingData['id']);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\OauthSetting' in method 'updateOauthSettings'. (undefined)

(StaticAccess)